client/src/use/useStyling.ts

```diff
@@ -59,38 +61,46 @@ export default function useStyling({ markChangesPending }: UseStylingParams) {
 
   function loadTypeColors(list?: Record<string, string>) {
     if (list) {
-    // Copy over the item so they can be modified in future
+      // Copy over the item so they can be modified in future
```

Comment:
```suggestion
      // Copy over the item so they can be modified in future
     customColors.value = {};
```
So there appears to be some reactivity shenanigans going on here.  The checklist items in `TypeList, TrackList` are mounted and rendered before the Color data is loaded from save so they reference an empty object.  The loading process doesn't seem to call reactivity on the checkboxes.  (Change colors -> save -> refresh, all items are correct except for the checkboxes until a redraw).  Setting the customColors to an empty object here seems to wake up the checkboxes into seeing that data is changed once it is loaded.

Reply:
Good catch.  Your fix will produce the right side-effect but the more correct fix is e865f13

Reply:
Also  33cce73 :)